### DISCUSSION TITLE
Change Log Level Idea

### DISCUSSION THREAD

**Discussion by televisi** [KBlixt/subcleaner#8] [Ideas]
Hi,
Thanks for creating such as great script!

Just an idea, how about the option to change the log level? 
E.g. to hide '[WARNING]' from the log and only show the '[INFO]'

Thanks

**Comment by KBlixt** [comment]
Yep, The logging needs a rewrite. it was kind of slapped on at the end without using proper tools. 

I'll redo this in my next version that have a few other goals in mind.

Glad you enjoy the script :)
